test/integration/Highlight.e2e.test.js

```diff
@@ -36,4 +36,29 @@ describe('Highlights', () => {
         // Assert that annotation target is active
         cy.get('.ba-HighlightTarget').should('have.class', 'is-active');
     });
+
+    it('should create a new highlight via promotion on a document', () => {
+        // Show the preview
+        cy.showPreview(Cypress.env('FILE_ID_DOC'));
+
+        // Wait for the highlight layer to be present
+        cy.getByTestId('ba-Layer--highlight');
+
+        // Assert that only one highlight annotation created in the above test is present
+        cy.get('.ba-HighlightTarget').should('have.length', 1);
```

Comment:
Can cypress tests be run individually? Seems like we should try to avoid relying on side effects of other tests if possible

Reply:
We would need to manually delete any existing annotations in a teardown step after each test and/or suite.

Reply:
That or we could go back to that logic @mxiao6 had to count the annotations before and after so it's more of a relative check?

Reply:
How about combining them into one test?